### PR TITLE
6.6.3: Add release notes

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds release notes for Sensu Go 6.6.3

## Motivation and Context
Support 6.6.3 patch release

### CODE REVIEW

**Review by echlebek — APPROVED** [review]

File: content/sensu-go/6.6/release-notes.md [hunk 1]
```diff
@@ -98,6 +99,29 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+**December 15, 2021** &mdash; The latest release of Sensu Go, version 6.6.3, is now available for download.
+
+Sensu Go 6.6.3 includes improvements to reduce load on clusters and support cluster recovery, as well as a backend configuration flag for specifying the internal etcd client log level. Fixes in this patch release will help prevent backend crashes when keepalive leases are revoked and when the backend cannot write to the event log file. In addition, this patch fixes issues that could result in a leaked etcd lease and keep the backend from terminating correctly.
+
+Read the [upgrade guide][1] to upgrade Sensu to version 6.6.3.
+
+**IMPROVEMENTS**
+
+- ([Commercial feature][259]) In the web UI, the default polling interval on the [entities page][261] is now 30 seconds to help reduce load on clusters.
```
Comment (portertech) [inline content/sensu-go/6.6/release-notes.md]: Missing the entities list view optimizations. ping @jamesdphillips

Comment (hillaryfraley) [inline content/sensu-go/6.6/release-notes.md]:
```suggestion
- ([Commercial feature][259]) In the web UI, the default polling interval on the [entities page][261] is now 30 seconds to help reduce load on clusters. Also, the web UI response time and memory usage is substantially improved when opening the entities page in the default state (loading the first page of results, with no search filter applied).
```

File: content/sensu-go/6.6/release-notes.md [hunk 2]
```diff
@@ -98,6 +99,29 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.6.3 release notes
+
+**December 15, 2021** &mdash; The latest release of Sensu Go, version 6.6.3, is now available for download.
```
Comment (hillaryfraley) [inline content/sensu-go/6.6/release-notes.md]:
```suggestion
**December 16, 2021** &mdash; The latest release of Sensu Go, version 6.6.3, is now available for download.
```

File: content/sensu-go/6.6/release-notes.md [hunk 3]
```diff
@@ -98,6 +99,29 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- The agentd daemon now starts up after all other daemons, which improves cluster recovery after the loss of a backend.
+- When using external etcd (the no-embed-etcd backend configuration flag is set to `true`), sensu-backend now crashes when its daemons do not stop within 30 seconds, which can happen due to an intentional shutdown or when database unavailability triggers an internal restart.
+When using embedded etcd, sensu-backend will still try to avoid crashing to prevent member corruption.
+
+**FIXES**
+
+- New agent sessions no longer result in a leaked etcd lease.
+- sensu-backend now prints a warning and continues instead of crashing when it cannot write to the specified event-log-file.
+- Fixed a bug that could cause a crash when keepalive leases are revoked on another backend or by an etcd operator.
+- Fixed an issue that could prevent sensu-backend from terminating correctly.
```
Comment (hillaryfraley) [inline content/sensu-go/6.6/release-notes.md:123]:
```suggestion
- Fixed an issue that could prevent sensu-backend from terminating correctly.
- Proxy entity state is now created when it is missing and a matching entity config already exists.
```

**Review by echlebek — APPROVED** [review]

**Pull request merged**